agent/instrumentation/micrometer-1.0/build.gradle.kts

```diff
@@ -42,6 +42,6 @@ dependencies {
 
 tasks {
   withType<Test>().configureEach {
-    jvmArgs("-Dapplicationinsights.internal.micrometer.step.millis=100")
+    jvmArgs("-Dapplicationinsights.internal.micrometer.step.millis=1000")
```

Comment:
this was the real fix for the MicrometerTest, I couldn't isolate why though as I was never able to reproduce locally, and the logging I added in CI pointed to some behavior deep in the micrometer internals